### DISCUSSION TITLE
Skip Transcoding only for Blu-Rays and UHD Blu-Rays

### DISCUSSION THREAD

**Discussion by jonasrichardrichter** [automatic-ripping-machine/automatic-ripping-machine#1253] [Q&A]
Hey 👋,

is there a way to skip transcoding only for BD and UHD BD? I want to transcode DVDs und CDs, but not BD.

**Comment by microtechno9000** [comment]
Not specifically, we are working on improving that at the moment 

Using sessions, users will be able to define based on media type or drive
